.github/workflows/ci.yml

```
@@ -202,12 +202,33 @@ jobs:
             bazel build --config=ci \
               --platforms=@io_bazel_rules_go//go/toolchain:${OS}_${ARCH} \
               //src/cmd/cloud-update:cloud-update \
-              --workspace_status_command="echo BUILD_VERSION ${{ steps.version.outputs.version }}"
+              --workspace_status_command="echo BUILD_VERSION ${{ needs.build.outputs.version }}"
             
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)